src/main/kotlin/no/nav/syfo/personstatus/application/IPersonOversiktStatusRepository.kt

```diff
@@ -19,7 +19,7 @@ interface IPersonOversiktStatusRepository {
 
     fun upsertSenOppfolgingKandidat(personident: PersonIdent, isAktivKandidat: Boolean): Result<Int>
 
-    fun upsertKartleggingssporsmalVurdering(personident: PersonIdent, isAktivVurdering: Boolean): Result<Int>
+    fun upsertKartleggingssporsmalKandidatStatus(personident: PersonIdent, isAktivKandidat: Boolean): Result<Int>
```

Comment:
Tanker om navneendring?

Reply:
Det er jo også mulig å endre navn på databasekolonnen. Tanken bak `aktiv vurdering` var at det er mer generelt. I hodet mitt så vurderer man kandidaten basert på svarene på kartleggingsspørsmålene, og at det er en vurdering som er aktiv og ikke en kandidat.
Med det sagt så er jeg helt åpen for å endre om dere andre tenker det er best 😊